app/controllers/assessments_controller.rb

```diff
@@ -217,12 +224,33 @@ def update
       # Monitorees can't edit their own assessments, so the last person to touch this assessment was current_user
       assessment.who_reported = current_user.email
 
+      old_reported_at = assessment.reported_at
+
+      if params.permit(:reported_at)[:reported_at].present?
+        # Parse as a Time object first to capture timezone for history comment formatting
+        begin
+          rep_time = Time.strptime(params[:reported_at], '%Y-%m-%d %H:%M %z')
+          assessment.reported_at = rep_time.to_datetime.utc
```

Comment:
Instead of converting to a DateTime, why not just parse the incoming parameter as a DateTime or just leave it as a Time object?

```ruby
report_time = DateTime.strptime(params[:reported_at], '%Y-%m-%d %H:%M %z')
assessment.reported_at = report_time.utc

# then further below...DateTime gives access to DateTime#zone
"#{old_reported_at.getlocal(rep_time.zone)) ..."

```